x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/privileged_access_detection/pad_routes.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { API_VERSIONS } from '../../../../../../common/entity_analytics/constants';
+import { useKibana } from '../../../../../common/lib/kibana';
+import type { GetStatusPrivilegedAccessDetectionPackageResponse } from '../../../../../../common/api/entity_analytics/privilege_monitoring/privileged_access_detection/status.gen';
+
+export const usePrivilegedAccessDetectionRoutes = () => {
+  const http = useKibana().services.http;
+
+  return useMemo(() => {
+    const getPrivilegedAccessDetectionStatus = async () => {
+      return http.fetch<GetStatusPrivilegedAccessDetectionPackageResponse>(
+        '/api/entity_analytics/privileged_user_monitoring/pad/status',
+        {
+          method: 'GET',
+          version: API_VERSIONS.public.v1,
+        }
+      );
+    };
+
+    const installPrivilegedAccessDetectionPackage = async () => {
+      return http.fetch('/api/entity_analytics/privileged_user_monitoring/pad/install', {
+        method: 'POST',
+        version: API_VERSIONS.public.v1,
+      });
+    };
+
+    const setupPrivilegedAccessDetectionMlModule = async () => {
+      return http.fetch('/internal/ml/modules/setup/pad-ml', {
+        version: '1',
+        method: 'POST',
+        body: JSON.stringify({
+          indexPatternName:
+            'logs-*,ml_okta_multiple_user_sessions_pad.all,ml_windows_privilege_type_pad.all',
```

Comment:
shoudl this default index pattern be in a const somwewhere? just seems a bit hidden like this, not 100% here